Images for extract/clip by extent (#7627)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectoroverlay.rst

Any features which intersect the extent will be included.

.. figure:: img/extractbyextent.png
  :align: center

>   Extract operation between a three-feature input layer 'a' and the extent, 
>   displayed in red for reference, of a two-feature overlay layer 'b' (left) - 

Reviewer: Is the layer b necessary? There are so many ways to set an extent that we could just use an extent
```suggestion
  Extract operation between a three-feature input layer 'a' and an extent, 
  displayed in red for reference - 
```

Author: You are probably right. I struggled with this a little. I couldn't think of another way to show where the extent was coming from. I deal with a lot of people new to GIS and the extent/bounding box concept is not obvious to them. In this case it is essentially a non visible geometry that is operating on their layer so I think I had the urge to show how bounding boxes work at the same time.

>   resulting features with dashed extent for reference (right)

Reviewer: should we keep showing the extent (the operation seems obvious to me without it but...)? No risk to think it's part of output? And what about keeping it red if we still display it?
